pycloudlib/lxd/tests/test_instance.py

```diff
@@ -36,12 +39,33 @@ def test_exec_with_run_command_on_xenial_machine(
             None, execute_via_ssh=False, series="xenial")
 
         instance._run_command(["test"], None)
-        expected_msg = (
-            "Many xenial images do not support executing commands"
-            " via exec due to missing kernel support: you may see"
-            " unavoidable failures.\nSee"
-            " https://github.com/canonical/pycloudlib/issues/132 for"
-            " details."
-        )
-        assert expected_msg in caplog.messages
+        assert any(self._missing_agent_msg in L for L in caplog.messages)
```

Comment:
Could this be simplified to `assert self._missing_agent_msg in caplog.text`? (This applies below, too.)

Reply:
Not really, as `caplog.text` is a list of strings and `self._missing_agent_msg in caplog.text` check if `self._missing_agent_msg` matches one of the list elements (exactly), while we want to to do a substring match here.

Not saying there isn't a smarter way to do so :-)

Reply:
I mixed up `.messages` and `.text`, your suggestion works. Thanks!